packages/DApp/src/hooks/useTimeLeft.ts

```diff
@@ -0,0 +1,15 @@
+import { useEffect, useState } from 'react'
+
+export function useTimeLeft(timeEndAt: number): number {
+  const [timeLeft, setTimeLeft] = useState(timeEndAt - Math.floor(Date.now() / 1000))
+
+  useEffect(() => {
+    const timer = setInterval(() => setTimeLeft(timeEndAt - Math.floor(Date.now() / 1000)), 1000)
+
+    return () => {
+      if (timer) clearInterval(timer)
```

Comment:
non-blockig: `timer` would be always defined, no

Reply:
addresed https://github.com/status-im/community-dapp/pull/88/commits/ff3a9468526c5598655df46d43a14632366b0bd1